src/CTA.Rules.Actions/ClassActions.cs

```diff
@@ -152,5 +151,100 @@ public class ClassActions
             };
             return RenameClass;
         }
+        public Func<SyntaxGenerator, ClassDeclarationSyntax, ClassDeclarationSyntax> GetReplaceMethodModifiersAction(string methodName, string modifiers)
+        {
+            Func<SyntaxGenerator, ClassDeclarationSyntax, ClassDeclarationSyntax> ReplaceMethodModifiers = (SyntaxGenerator syntaxGenerator, ClassDeclarationSyntax node) =>
+            {
+                var allMembers = node.Members.ToList();
+                var allMethods = allMembers.OfType<MethodDeclarationSyntax>();
+                if (allMethods.Any())
+                {
+                    var replaceMethod = allMethods.FirstOrDefault(m => m.Identifier.ToString() == methodName);
+                    if (replaceMethod != null)
+                    {
+                        var allModifiersAreValid = modifiers.Split(new char[] { ' ', ',' }).All(m => Constants.SupportedMethodModifiers.Contains(m));
+                        if (allModifiersAreValid)
+                        {
+                            SyntaxTokenList tokenList = new SyntaxTokenList(SyntaxFactory.ParseTokens(modifiers));
+                            var newMethod = replaceMethod.WithModifiers(tokenList);
+
+                            node = node.WithMembers(node.Members.Replace(replaceMethod, newMethod)).NormalizeWhitespace();
+                        }
+                    }
+                }
+
+                return node;
+            };
+            return ReplaceMethodModifiers;
+        }
+        public Func<SyntaxGenerator, ClassDeclarationSyntax, ClassDeclarationSyntax> GetAddGlobalExpressionAction(string expression)
+        {
+            Func<SyntaxGenerator, ClassDeclarationSyntax, ClassDeclarationSyntax> RenameClass = (SyntaxGenerator syntaxGenerator, ClassDeclarationSyntax node) =>
+            {
+                MemberDeclarationSyntax parsedExpression = SyntaxFactory.ParseMemberDeclaration(expression);
+                if (!parsedExpression.FullSpan.IsEmpty)
+                {
+                    var nodeDeclarations = node.Members;
+                    nodeDeclarations = nodeDeclarations.Insert(0,parsedExpression);
+                    node = node.WithMembers(nodeDeclarations).NormalizeWhitespace();
+                }
+                return node;
+            };
+            return RenameClass;
+        }
```

Comment:
Two comments here:

1. Would it make sense to rename this function to `GetAddExpressionAction`, similar to what's in `MethodDeclarationActions.cs`? My thought is that `GlobalExpression` implies global scope, but this action permits the user to add expressions that are limited to just the class as well.

2. Should `RenameClass` be renamed to something like `AddGlobalExpression`?

Reply:
1. Agreed, renamed as mentioned.
2. I would argue that rename class is straightforward and would prefer to leave it as is. Mark checked in that action.